spring-cloud-app-broker-core/src/main/java/org/springframework/cloud/appbroker/manager/BackingAppManagementService.java

```diff
@@ -82,8 +114,24 @@ public BackingAppManagementService(ManagementClient managementClient, AppDeploye
 			.then();
 	}
 
+	/**
+	 * Helper method that fetches service name and plan name from Cloud Foundry Service Instances API (CF API) and
+	 * invokes {@code restart(serviceInstanceId, serviceName, planName)}.
+	 *
+	 * Because this method will try to fetch user-created service instance, UAA client used by the broker has to be a
+	 * space developer and an org user of the target space and org, or has to have {@code cloud_controller.admin}
```

Comment:
could 'target space and org' be confused with the backing app target space and org? Also tempted to drop the org user bit since it's implied by space developer. Wonder if we should reword slightly to something like 'the space containing the service instance'